README.rst

```diff
@@ -253,118 +253,11 @@ Please note that this project is released with a Contributor Code of Conduct.
-and `PositiveSmallIntegerField
-<https://docs.djangoproject.com/en/3.2/ref/models/fields/#positivesmallintegerfield>`__
-are both stored as `Integer type
-<https://cloud.google.com/spanner/docs/data-types#integer_type>`__
-.
-
-``Meta.order_with_respect_to`` model option isn't supported
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-
-This feature uses a column name that starts with an underscore
-(``_order``) which Spanner doesn't allow.
-
-Random ``QuerySet`` ordering isn't supported
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-
-Spanner does not support it and will throw an exception. For example:
-
-::
-
-    >>> ExampleModel.objects.order_by('?')
-    ...
-    django.db.utils.ProgrammingError: 400 Function not found: RANDOM ... FROM
-    example_model ORDER BY RANDOM() ASC
-
-Schema migrations
-~~~~~~~~~~~~~~~~~
-
-There are some limitations on schema changes to consider:
-
--  No support for renaming tables and columns;
--  A column's type can't be changed;
--  A table's primary key can't be altered.
-
-``DurationField`` arithmetic doesn't work with ``DateField`` values (`#253 <https://github.com/googleapis/python-spanner-django/issues/253>`__)
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-
-Spanner requires using different functions for arithmetic depending on
-the column type:
-
--  ``TIMESTAMP`` columns (``DateTimeField``) require ``TIMESTAMP_ADD``
-   or ``TIMESTAMP_SUB``
--  ``DATE`` columns (``DateField``) require ``DATE_ADD`` or ``DATE_SUB``
-
-Django does not provide ways to determine which database function to
-use. ``DatabaseOperations.combine_duration_expression()`` arbitrarily uses
-``TIMESTAMP_ADD`` and ``TIMESTAMP_SUB``. Therefore, if you use a
-``DateField`` in a ``DurationField`` expression, you'll likely see an error
-such as:
-
-::
-
-    "No matching signature for function TIMESTAMP\_ADD for argument types:
-    DATE, INTERVAL INT64 DATE\_TIME\_PART."
-
-Computations that yield FLOAT64 values cannot be assigned to INT64 columns
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-
-Spanner does not support this (`#331
-<https://github.com/googleapis/python-spanner-django/issues/331>`__) and will
-throw an error:
-
-::
-
-    >>> ExampleModel.objects.update(integer=F('integer') / 2)
-    ...
-    django.db.utils.ProgrammingError: 400 Value of type FLOAT64 cannot be
-    assigned to integer, which has type INT64 [at 1:46]\nUPDATE
-    example_model SET integer = (example_model.integer /...
-
-Addition with null values crash
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-
-Additions cannot include ``None`` values. For example:
+LIMITATIONS
+-----------------
 
-::
+Spanner has certain limitations of it's own and a full set of limitations are documented over `here <https://cloud.google.com/spanner/quotas#schema_limits>`_
```

Comment:
Autofield is not covered in this documentation. Is that something that should be added separately?